### ISSUE TITLE
How many epochs did you train?

### ISSUE THREAD

**Issue by lee-man** [nrupatunga/goturn-pytorch#6]
Hi,

Thanks for your excellent code. I just wanna check how many epochs did you train?
3 epochs?
I am trying to train the model myself and it seems like it will take about 2 days for a single epoch.

Thanks!

**Comment by nrupatunga** [comment]
Hi, 

I trained on 1080Ti, It should train within a day. 
What is data loader speed, how many iters/sec, does it show in the progress bar.

**Comment by lee-man** [comment]
It took about 5 seconds per iteration... I will check the setting. Thanks for your reply.

**Comment by nrupatunga** [comment]
This figure also gives you an idea. (look at the x-axis)

https://github.com/nrupatunga/goturn-pytorch/blob/master/doc/images/loss.png

Also, data on SSD drive might speedup, though I remember my data was on normal HDD.
